src/components/organisms/footer/footer.html

```diff
@@ -0,0 +1,37 @@
+<div class="footer">
+  <div class="footer-body">
+    <img src="../../../img/footer.png" class="footer-img" />
+    <div class="information">
+      〒532-0011　大阪市淀川区西中島6丁目8-23テスト<br />
+      TEL.06-1234-5678 FAX.06-1234-5678　営業時間 / 9:00 - 22:00　定休日 /
+      不定休
+      <ul class="icons">
+        <li class="list-item">
+          <a href="#" class="footer-icon"><i class="fab fa-facebook-f"></i></a>
+        </li>
+        <li class="list-item">
+          <a href="#" class="footer-icon"><i class="fab fa-twitter"></i></a>
+        </li>
+        <li class="list-item">
+          <a href="#" class="footer-icon"><i class="fab fa-instagram"></i></a>
+        </li>
+        <li class="list-item">
+          <a href="#" class="footer-icon"><i class="fas fa-rss"></i></a>
+        </li>
+      </ul>
+    </div>
+  </div>
+  <div class="footer-menu">
+    <a href="#" class="footer-menu-item">HOME</a>
+    <a href="#" class="footer-menu-item">CONCEPT</a>
+    <a href="#" class="footer-menu-item">SERVICE</a>
+    <a href="#" class="footer-menu-item">FLOOR</a>
+    <a href="#" class="footer-menu-item">INTERVIEW</a>
+    <a href="#" class="footer-menu-item">BLOG</a>
+    <a href="#" class="footer-menu-item">ACCESS</a>
+    <a href="#" class="footer-menu-item">DOWNLOAD</a>
+  </div>
+  <div class="credit">
```

Comment:
```suggestion
  <small class="credit">
```
細かいですがこのようにもかけますね
https://developer.mozilla.org/ja/docs/Web/HTML/Element/small